index.html

```diff
@@ -0,0 +1,369 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <link rel="stylesheet" href="index.css">
+  <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+  <meta charset="utf-8">
+  <title>DESIGN TEARDOWN</title>
+</head>
+
+<body>
+
+  <header>
+    <nav>
+      <div class="headerBox">
+        <h3>Smashing Magazine</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Articles</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Books</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Events</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Jobs</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>Membership</h3>
+      </div>
+
+      <div class="headerBox">
+        <h3>TOPICS</h3>
+      </div>
+    </nav>
+
+  </header>
+
+
+  <main>
+    <section id="firstSection">
```

Comment:
`firstSection` is half-width container :mark: 
- [ ] Each container should have an image in the top left corner with the title of an article under it.